specific/cmo_project/models/hr.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from openerp import fields, models
+
+
+class HrDepartment(models.Model):
+    _inherit = 'hr.department'
+
+    active = fields.Boolean(
+        string='Active',
+        default=False,
```

Comment:
Default = False?, so when the new department is created manually, it can't be used unit someone set it to True, right?